notebooks/samples/explanations/ai-explanations-image-tf2.ipynb

```diff
@@ -0,0 +1,1758 @@
+{
```

Comment:
<p>Suggestions to [avoid using the word "may"](<a href="https://developers.google.com/style/word-list#may" target="_blank">https://developers.google.com/style/word-list#may</a>):</p><p> </p><p>You can also change...</p><p> </p><p>Do not use...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/ai-platform-samples/pull/192/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>